upstairs/src/test.rs

```diff
@@ -472,69 +472,85 @@ pub(crate) mod up_test {
         Ok(())
     }
 
-    #[test]
-    fn send_io_live_repair_read() {
+    fn test_ddef(blocks_per_extent: u64) -> RegionDefinition {
         // Check the send_io_live_repair for a read below extent limit,
```

Comment:
I think this comments belongs with `send_io_live_repair_read()` test below.

Reply:
Good catch, fixed!